pkg/result/result.go

```diff
@@ -228,24 +228,22 @@ func summarize(status types.MisconfStatus, summary *types.MisconfSummary) {
 
 func applyPolicy(ctx context.Context, vulns []types.DetectedVulnerability, misconfs []types.DetectedMisconfiguration,
 	policyFile string) ([]types.DetectedVulnerability, []types.DetectedMisconfiguration, error) {
-	policy, err := os.ReadFile(policyFile)
-	if err != nil {
-		return nil, nil, xerrors.Errorf("unable to read the policy file: %w", err)
-	}
+	var err error
+	var policyStore PolicyStore
 
-	query, err := rego.New(
-		rego.Query("data.trivy.ignore"),
-		rego.Module("lib.rego", module),
-		rego.Module("trivy.rego", string(policy)),
-	).PrepareForEval(ctx)
+	if utils.IsValidURL(policyFile) {
+		policyStore, err = NewRemotePolicyStore(policyFile)
```

Comment:
Can we add a test case for RemotPolicyStore in result_test.go? I think we can use httptest.